plugins/acss-kit/commands/kit-create.md

```
@@ -0,0 +1,55 @@
+description: Generate any acss-kit component from a natural-language description (creator mode)
+argument-hint: <description>
+allowed-tools: Read, Glob, Grep, Write, Edit, Bash, AskUserQuestion
+---
+
+# /kit-create
+
+Creator mode — describe a UI element in plain English and have it generated as a paste-ready snippet (or standalone component file). Works with any component listed in `references/components/catalog.md` (Button, Alert, Card, Dialog, Badge, Link, Input, Field, Checkbox, IconButton, Img, Icon, List, Table, Popover, Nav).
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)